### DISCUSSION TITLE
Recommended way to update theme and npm dependencies

### DISCUSSION THREAD

**Discussion by LITUATUI** [hbstack/theme-cards#264] [Q&A] [answered]
What's the recommended way to update the theme and the npm dependencies?

I usually do this from the terminal:

`hugo mod get -u ./... && hugo mod npm pack && npm update && hugo mod tidy`

Is this the best way? Thanks!

**Comment by razonyang** [comment, marked as answer]
This theme doesn't use NPM to manage dependencies except the build tools. In general, update the modules is good enough.

```sh
hugo mod get -u ./... && hugo mod tidy
```

You can update the NPM build tools monthly or even longer.

```sh
npm update
```

But it depends, if you rely on other non-HB modules, please refer to their docs.
